### PR TITLE
added initial documentation for adding new model

### PR DESCRIPTION
**Pull request by Honaker**
I added documentation to show 3 different examples of how to bring your own model into an armory-library example.
-Using a model from HuggingFace
-Using a model from GitHub
-Using a model from a python library in PyPI

### CODE REVIEW

File: docs/developers/new_model_to_armory.md [hunk 1]
```diff
@@ -0,0 +1,109 @@
...
+    loss=torch.nn.CrossEntropyLoss(),
+    optimizer=torch.optim.Adam(model.parameters(), lr=0.003),
+    input_shape=(3, 224, 224),
+    channels_first=True,
+    nb_classes=150,
+    clip_values=(-1, 1),
+)
+```
+For this section of code we create two variables model and classifier. We found a model on HuggingFace that is trained on the same pokemon image dataset with the model card 'tianzhihui-isc/vit-base-patch16-224-in21k-finetuned-pokemon-classification'. This can be replaced with another model off of Huggingface.
+- The model variable uses AutoModelForImageClassification.from_pretrained which takes in a huggingface model card name as a variable. This retrieves the model from
```
Comment (treubig26) [inline docs/developers/new_model_to_armory.md:36]:
```suggestion
- The model variable uses `AutoModelForImageClassification.from_pretrained` which takes in a HuggingFace model card name as a variable. This retrieves the model from
```

File: docs/developers/new_model_to_armory.md [hunk 2]
```diff
@@ -0,0 +1,109 @@
...
+    HuggingFace that we will use for this example. `track_params` is a function wrapper that stores the argument values as parameters in MLflow. Lastly,
+    the JaticImageClassificationModel is another wrapper to make the model compatible with Armory. This allows the model to have a standard output like other
```
Comment (treubig26) [inline docs/developers/new_model_to_armory.md:38]:
```suggestion
    the `JaticImageClassificationModel` is another wrapper to make the model compatible with Armory. This allows the model to have a standard output like other
```

File: docs/developers/new_model_to_armory.md [hunk 3]
```diff
@@ -0,0 +1,109 @@
...
+    jatic image classification models.
```
Comment (treubig26) [inline docs/developers/new_model_to_armory.md:39]:
```suggestion
    JATIC image classification models.
```

File: docs/developers/new_model_to_armory.md [hunk 4]
```diff
@@ -0,0 +1,109 @@
...
+- The `PyTorchClassifier` class wraps the model to be usable by the ART library. It is specific to image classifier models written within the PyTorch framework. It takes in as arguments the model, loss function, and optimizer. The input image sizes are the shape of all the images inside the dataset. The channels_first variable is true because the images in the pokemon dataset are in a channels-first (C, H, W) multi-dimensional array. The nb_classes describe the number of classes model predicts on. Lastly the clip value  is the values that will be the min and max values of the input after scaling.We use `track_init_params` so that the constructor parameters for the ART wrapper are also tracked in MLflow.
```
Comment (treubig26) [inline docs/developers/new_model_to_armory.md:40]:
```suggestion
- The `PyTorchClassifier` class wraps the model to be usable by the ART library. It is specific to image classifier models written within the PyTorch framework. It takes in as arguments the model, loss function, and optimizer. The input image sizes are the shape of all the images inside the dataset. The `channels_first` variable is true because the images in the pokemon dataset are in a channels-first (C, H, W) multi-dimensional array. The `nb_classes` describe the number of classes model predicts on. Lastly the clip value  is the values that will be the min and max values of the input after scaling.We use `track_init_params` so that the constructor parameters for the ART wrapper are also tracked in MLflow.
```

File: docs/developers/new_model_to_armory.md [hunk 5]
```diff
@@ -0,0 +1,109 @@
...
+    model,
+    loss=torch.nn.CrossEntropyLoss(),
+    optimizer=torch.optim.Adam(model.parameters(), lr=0.003),
+    input_shape=(3, 224, 224),
+    channels_first=True,
+    nb_classes=150,
+    clip_values=(-1, 1),
+)
+```
+For this section of code we create two variables model and classifier. We found a model on HuggingFace that is trained on the same pokemon image dataset with the model card 'tianzhihui-isc/vit-base-patch16-224-in21k-finetuned-pokemon-classification'. This can be replaced with another model off of Huggingface.
```
[the remark this answers is not in the document]
Comment (Honaker) [reply docs/developers/new_model_to_armory.md:35]: Thanks for help! I got this from the other PR for the colab file since I am running the same lines as that example. I can make these changes in that PR as well b/c the same issues should be there.

File: docs/developers/new_model_to_armory.md [hunk 6]
```diff
@@ -0,0 +1,105 @@
...
+from efficientnet_lite0_pytorch_model import EfficientnetLite0ModelFile
+weights_path = EfficientnetLite0ModelFile.get_model_file_path()
+
+lite0_model = EfficientNet.from_pretrained('efficientnet-lite0', weights_path = weights_path )
+```
+
+This is the same code from the last two examples to create a model and classifier variables.
+```python
+model = JaticImageClassificationModel(  
+    SRRmodel
```
Comment (lori101) [inline docs/developers/new_model_to_armory.md:92]: looks like this should be lite0_model, not SRRmodel

**Review by mwartell — APPROVED** [review]
